{{ cookiecutter.project_slug }}/locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,166 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-10-25 22:53+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: {{ cookiecutter.project_slug }}/agents/templates/notifications/agent_created_notification_email.html:5
+msgid "New agent created"
+msgstr "Nueva agente creada"
+
+#: {{ cookiecutter.project_slug }}/agents/templates/notifications/agent_created_notification_email.html:10
+#, python-format
+msgid ""
+"\n"
+"You're receiving this email because you are an administrator and a new\n"
+"agent has been created.\n"
+"\n"
+"Created By: %(created_by)s\n"
```

Comment:
For reference, `%(created_by)s` is in Django's named-string interpolation format. None of the strings in django.po require the use of pluralization.